.github/workflows/e2e-main.yml

```diff
@@ -33,7 +33,8 @@ jobs:
         # 23 is failing ? spec(s)
         # 31 is failing ? spec(s)
         # 24 is currently failing all the time, disabling it for now
-        android-api-level: [30]
+        # 30 is not included as it runs on the merge queue
+        android-api-level: []
```

Comment:
Does this work with an empty array?

Reply:
good call. It doesn't. commented the entire job for now